azext_edge/edge/commands_dataflow.py

```diff
@@ -36,6 +36,40 @@ def list_dataflows(cmd, profile_name: str, instance_name: str, resource_group_na
     )
 
 
+def apply_dataflow_endpoint(
+    cmd,
+    endpoint_name: str,
+    instance_name: str,
+    resource_group_name: str,
+    config_file: str,
+    **kwargs,
+) -> dict:
+    return DataFlowEndpoints(cmd).apply(
+        name=endpoint_name,
+        instance_name=instance_name,
+        resource_group_name=resource_group_name,
+        config_file=config_file,
+        **kwargs,
+    )
+
+
+def delete_dataflow_endpoint(
+    cmd,
+    endpoint_name: str,
+    instance_name: str,
+    resource_group_name: str,
+    confirm_yes: bool = False,
+    **kwargs,
+) -> None:
+    DataFlowEndpoints(cmd).delete(
```

Comment:
nit, this could still `return` even though it doesn't return a payload, to be in line with other deletes. plus the `DataFlowEndpoints(cmd).delete()` implementation returns.